notebooks/tel_and_site/subsys_req_ver/tma/test_star_selector.ipynb

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
I don't quite understand what this notebook does that the other one doesn't.

Some, text at the top of this notebook too would be useful. Or maybe this one isn't necessary?

Reply:
Indeed, I forgot to push the commit that deleted this, thanks.